WalletWasabi.Gui/Tabs/WalletManager/LoadWalletView.xaml

```diff
@@ -1,28 +1,46 @@
 <UserControl xmlns="https://github.com/avaloniaui"
             xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui">
   <DockPanel LastChildFill="True">
-    <StackPanel DockPanel.Dock="Bottom" Margin="20 10">
-      <DockPanel LastChildFill="True">
-        <Button Command="{Binding LoadCommand}" DockPanel.Dock="Right">
-          <StackPanel Orientation="Horizontal">
-            <controls:Spinner Height="20" Width="20" IsVisible="{Binding IsBusy}" Margin="0 0 8 0" />
-            <TextBlock Text="{Binding LoadButtonText}" />
+    <StackPanel DockPanel.Dock="Bottom" Spacing="10">
+      <controls:GroupBox IsVisible="{Binding RequirePassword}" TextBlock.FontSize="30" Padding="30" Margin="20 0">
+        <Grid Classes="content">
+          <StackPanel Spacing="8">
+            <StackPanel>
+              <TextBlock Text="Select a wallet to test its password!" FontWeight="Bold" />
+            </StackPanel>
+            <controls:NoparaPasswordBox Password="{Binding Password}" Watermark="Password" UseFloatingWatermark="True" />
+            <StackPanel>
+              <TextBlock Text="Why the unconventional password masking?" FontWeight="Bold" />
+              <TextBlock Text="We received numerous reports on losing or mistyping passwords. This experiment aims to disrupt users' typical workflow at wallet generation, forcing them to pay more attention than otherwise would. Why use it in other places, too? Because nopara73 finds it funny." TextWrapping="Wrap" />
```

Comment:
I would remove the last question and the answer.

Reply:
Done.